provisioning/roles/cuttlefish-app/tasks/main.yml

```diff
@@ -92,3 +92,11 @@
     owner: root
     group: root
     mode: 0744
+
+- name: Create old email archiving daily cronjob for 1am
```

Comment:
> I've set this to run at 1am, which is an hour after the network and
CPU intensive backup job.

From the charts in linode, nighttime is a quieter than in the day. My thinking was that night time is a good time to run this then, so that if something goes wrong or the app crashes, not as many people will see it and we can get to it first thing. The backup runs at midnight and consumes heaps of memory for about 15/20 min. On the next hour is quiet and seemed like a good idea to do it before it's too late so someone could stay up or might be awake if it needs attention.